vignettes/art-contrasts.Rmd

```diff
@@ -14,19 +14,17 @@ vignette: >
 The aligned-rank transform (ART) allows for non-parametric analyses of variance. 
 But how should we do contrast tests with ART? 
 
-**Within a single factor**, contrasts in ART are straightforward: simply conduct contrasts on the linear model containing the response aligned for that factor.
+Contrasts involving levels of **any number of factors** can be conducted using the ART-C method.
```

Comment:
I'd rather a bit more explanation here since the issue is nuanced, and people familiar with the old method might be confused. Maybe add a short definition of ART-C ("a method for aligning-and-ranking to conduct contrasts with ART"?). 

I would also leave in something about their being different types of interaction contrasts, except now mentioning the fact that we can do both types :)

Maybe something mentioning that the

Reply:
also should explicitly mention somewhere (either here or on first usage) that ART-C is the method implemented by the `art.con()` function